services/director-v2/tests/unit/test_modules_dynamic_sidecar_scheduler.py

```diff
@@ -525,9 +529,9 @@ async def test_mark_all_services_in_wallet_for_removal(
         wallet_id = scheduler_data.wallet_info.wallet_id
         can_remove = scheduler_data.dynamic_sidecar.service_removal_state.can_remove
         match wallet_id:
-            case WalletID(1):
```

Comment:
match wallet_id:
>               case WalletID(1):
E               TypeError: called match pattern must be a type